sdk/synapse/azure-synapse-artifacts/tests/testcase.py

```diff
@@ -10,8 +10,6 @@
 
 
 class SynapseArtifactsTest(AzureTestCase):
```

Comment:
Ah, it looks like this is why pipeline tests are attempting to use `vcrpy` instead of the test proxy. Everything else looks correct, so I think inheriting from AzureRecordedTestCase should get things working:
```suggestion
class SynapseArtifactsTest(AzureRecordedTestCase):
```

Reply:
Thank you for your suggestion. Updated in https://github.com/Azure/azure-sdk-for-python/pull/25600/commits/59b62d1697453c9837f006372f22e3bfaf8cf1bb